src/main/java/Graphviz.java

```diff
@@ -272,7 +270,7 @@ private static void generateJsonModuleGraph(JsonObject module, File outputFolder
 
     File outputFile = outputFolder.toPath().resolve(relativePath + ".png").toFile();
     outputFile.mkdirs();
-    guru.nidi.graphviz.engine.Graphviz.fromGraph(g).rasterizer(Rasterizer.BATIK)
```

Comment:
I'm trying to remember why this was necessary in the first place... I think it may have been because the default rasterizer didn't work on CI? (Which was probably still Travis at the time) I do seem to recall that the default was a lot faster on my laptop though so if this still works on CI without it then that's a win. 
(Though I have to ask, is maintaining this graphviz support still necessary given that everybody uses the module builder?)

Reply:
> (Though I have to ask, is maintaining this graphviz support still necessary given that everybody uses the module builder?)

That depends on the definition of "necessary" and the definition of "everybody."

I think the static images are worthwhile. We post them in our gallery, which people still look at. The images can also be used for documentation/figures without resorting to manual screenshots.